README.md

```diff
@@ -46,12 +46,9 @@ CoinSwap is a rust implementation of a variant of atomic-swap protocol, using HT
 * [Developer's resources](/docs/dev-book.md)
 
 ## Build and Test
-Macos (M series) needs some feature for compilation 
-
 ```console
-$ cargo build --target=aarch64-apple-darwin  --features=build-macos-aarch
+$ cargo build
 ```
```

Comment:
remove this section

Reply:
so like its just the basic build cmd, we can keep also ig ?

Reply:
Thats the point just a basic build command.